<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1561522748074' stop='1561523190288'>
    <name>Login MyStake Application</name>
    <title>Login MyStake Application</title>
    <test-cases>
        <test-case start='1561522748075' status='failed' stop='1561523148001'>
            <name>Verify Successful Login</name>
            <title>Verify Successful Login</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='2770b16f-6498-4853-b79d-1691ec164a73-attachment.png' type='image/png' size='24979'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (/Users/Progressio/Documents/Protractor/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4281:23)
                    at ontimeout (timers.js:436:11)
                    at tryOnTimeout (timers.js:300:5)
                    at listOnTimeout (timers.js:263:5)
                    at Timer.processTimers (timers.js:223:10)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1561523148001' status='passed' stop='1561523190287'>
            <name>Verify Invalid Credentials</name>
            <title>Verify Invalid Credentials</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='6bf156a6-3803-4748-b5d4-6a8d2a405298-attachment.png' type='image/png' size='26648'/>
            </attachments>
        </test-case>
    </test-cases>
</ns2:test-suite>